public/css/icinga/base.less

```diff
@@ -39,6 +39,30 @@
 @icinga-blue-light: fade(@icinga-blue, 50%);
 @icinga-blue-dark: #0081a6;
 
+// ipl-web overrides
```

Comment:
We should override all variables I think. Not just a few. Our themes may only require a few, but what about others? We noticed that it might be useful to override them in Icinga Web 2, so please do it for all variables that are subject to this.

Reply:
Wouldn’t that make it wildly confusing? This way we’ll have multiple vars that stand for the same thing.

Wouldn’t it be easier to have all the vars renamed in icingaweb2 to the their ipl-web counterparts (at least where possible), so that they are overriden anyway?

This might be a little effort for now, but it could be worth in the long term.